docs/resources/mws_customer_managed_keys.md

```diff
@@ -64,6 +64,7 @@ resource "aws_kms_alias" "managed_services_customer_managed_key_alias" {
 }
 
 resource "databricks_mws_customer_managed_keys" "managed_services" {
+    provider     = databricks.mws
```

Comment:
please revert this file, as it has no provider definition with this alias. also it might be a separate module, so it's a completely valid doc.

```
provider "databricks" {
  alias = "mws"
}
```

Reply:
Done.